.github/workflows/create-release.yaml

```diff
@@ -47,4 +41,13 @@ jobs:
           prerelease: false
           body: ${{ steps.extract-release-notes.outputs.release_notes }}
         env:
-          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
+      - name: Publish packages to crates.io
```

Comment:
I think the name here isn't quite correct

Reply:
what should it be?